### DISCUSSION TITLE
having problem to start peacock

### DISCUSSION THREAD

**Discussion by ziyiwang141319** [idaholab/moose#25814] [Q&A Getting Started] [answered]
I am using mac system and following the instruction on https://mooseframework.inl.gov/python/peacock.html for peacock.
In the environmenpart part, I created a bash_profile and put:
        

 `export PATH=/Users/xuke/projects/moose/python/peacock:$PATH`

in bash_profile. 

And i want to try the example in moose, so i did 

```
(peacock) xuke@xukedeMac-mini ~ % cd ~/projects/moose/examples/ex01_inputfile
(peacock) xuke@xukedeMac-mini ex01_inputfile % peacock
zsh: command not found: peacock
```

I also tried this but still did not work
```
(peacock) xuke@xukedeMac-mini gui % pwd                                   
/Users/xuke/projects/moose/gui
(peacock) xuke@xukedeMac-mini gui % peacock
zsh: command not found: peacock

```
Here is the files in gui folder
```
(base) xuke@xukedeMac-mini gui % ls
BasePeacockMainWindow.py	PostprocessorViewer
CheckRequirements.py		PythonConsoleWidget.py
Execute				__init__.py
ExodusViewer			__pycache__
Input				base
LogWidget.py			icons
PeacockApp.py			peacock
PeacockException.py		tests
PeacockMainWindow.py		utils
```

I am unclear what to do to run peacock. Thanks for helping.

**Comment by milljm** [comment, marked as answer]
```pre
zsh: command not found: peacock
^^^ means you are using ZSH and not BASH
```
You need to instead modify your ~/.zshrc profile.

Edit: Those instructions are dated... we need to add `~/.zshrc`
Issue created: https://github.com/idaholab/moose/issues/25818

**Reply by milljm** [reply]
Oh my... those instructions are _very_ dated. You no longer need to operate in a separate `peacock` environment. If you have installed the latest `moose-dev`, this includes `moose-peacock`. If you would, please exit that peacock environment, and delete it :). You can safely use `peacock` while in the `(moose)` environment.

#### A little history:
The reason they were separated before, was due to all the dependencies involved were causing conflicts with the toolchain being installed with `moose-mpich`. This was remedied not to long ago with version `moose-dev=2023.09.19`.

**Reply by ziyiwang141319** [reply]
Thank you so much for helping. I can run peacock now.

**Reply by ankit72242** [reply]
Hello sir I'm facing issue with peacock gui 

The error is "Segmentation fault(core dumped)"
When try simply peacock command.

I have successfully installed the Moose update path done everything.
